### PR TITLE
feat: Super collapsed block

### PR DESCRIPTION
**Pull request by Ambrdctr**
Exemple de fonctionnement en mettant le message 3 puis le message 5 en non lu:
image1
image2

deps on this fix https://github.com/Infomaniak/ios-kMail/pull/1484

### CODE REVIEW

File: Mail/Views/Thread/SuperCollapsedView.swift [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        ZStack {
+            Rectangle()
+                .strokeBorder(MailResourcesAsset.textFieldColor.swiftUIColor, lineWidth: 1)
+                .background {
+                    Rectangle()
+                        .fill(MailResourcesAsset.backgroundCardSelectedColor.swiftUIColor)
+                }
+                .frame(height: 8)
+
+            Button {
```
Comment (valentinperignon) [inline Mail/Views/Thread/SuperCollapsedView.swift:37]:
```suggestion
            Button(action: action) {
```

File: Mail/Views/Thread/SuperCollapsedView.swift [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                action()
+            } label: {
+                Text(MailResourcesStrings.Localizable.superCollapsedBlock(count))
+                    .textStyle(.bodyAccent)
+                    .padding(.vertical, 10)
+                    .padding(.horizontal, 20)
```
Comment (valentinperignon) [inline Mail/Views/Thread/SuperCollapsedView.swift:43]: Can we normalize the padding? Something like vertical 8 (UIPadding.small) and horizontal 16 or 24 (UIPadding.regular or medium)?

**Review by valentinperignon — COMMENTED** [review]
Just reviewed the UI part! Really great feature 🤩

**Review by PhilippeWeidmann — COMMENTED** [review]
Something is broken when loading a thread for the first time. Last message of the thread never loads.

**Review by adrien-coye — COMMENTED** [review]
I did check for the `initial loading` bug.  My understanding is that the processing on the last email is performed quickly, but does not redraw the view.  It relies on a binding. I'm experimenting with `InlineAttachmentWorker: ObservableObject` without any luck either. I'll check on Monday with @valentinperignon as we discussed.

**Review by adrien-coye — APPROVED** [review]
LGTM 🤝

**Pull request merged**